Sources/GoTrue/Internal/PKCE.swift

```diff
@@ -1,19 +1,20 @@
-import CryptoKit
+import Crypto
```

Comment:
It seemed more straightforward to have one dependency on Crypto which is open source and designed to be portable across operating systems.

Reply:
LGTM, thank you.